### PR TITLE
BUG: short-term hack to make spot detection more similar to cellprofiler

### PR DESCRIPTION
**Pull request by ambrosejcarr**
- Add a `radius_is_gyration` parameter to account for the divergent way that `trackpy.locate` and `skimage.blob_log` calculate radius. 
- fixes a bug that relates to exclusive indexing.

### CODE REVIEW

File: notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py [hunk 1]
```diff
@@ -210,10 +210,6 @@
 table = pd.Series(counts, index=genes).sort_values(ascending=False)
 # EPY: END code
 
-# EPY: START code
-table.head()
```
Comment (dganguli) [inline notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py]: un-nuke this

File: starfish/pipeline/features/spots/detector/detect.py [hunk 2]
```diff
@@ -29,6 +30,11 @@ def measure_spot_intensity(
         SpotAttributes table containing coordinates and radii of spots
     measurement_function : Callable[[Sequence], Number])
         Function to apply over the spot volumes to identify the intensity (e.g. max, mean, ...)
+    radius_is_gyration : bool
+        if True, indicates that the radius being passed is gyration, which is a function of
```
Comment (dganguli) [inline starfish/pipeline/features/spots/detector/detect.py]: "radius being passed is gyration" -> "radius corresponds to the radius of gyration"

File: starfish/pipeline/features/spots/detector/detect.py [hunk 3]
```diff
@@ -72,6 +81,11 @@ def measure_spot_intensities(
         Locations and radii of spots
     measurement_function : Callable[[Sequence], Number])
         Function to apply over the spot volumes to identify the intensity (e.g. max, mean, ...)
+    radius_is_gyration : bool
```
Comment (dganguli) [inline starfish/pipeline/features/spots/detector/detect.py]: copy doc-string from measure_spot_intensity

File: starfish/pipeline/features/spots/detector/detect.py [hunk 4]
```diff
@@ -176,6 +195,11 @@ def detect_spots(
         rounds found in data_image.
     measurement_function : Callable[[Sequence], Number]
         the function to apply over the spot area to extract the intensity value (default 'np.max')
+    radius_is_gyration : bool
```
Comment (dganguli) [inline starfish/pipeline/features/spots/detector/detect.py]: see above

File: notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py (hunk 1 above)
Comment (ambrosejcarr) [reply notebooks/ISS_Pipeline_-_Breast_-_1_FOV.py]: It is present directly below, this is repetitive.

**Pull request merged**